kuma/search/filters.py

```diff
@@ -192,3 +194,21 @@ def filter_queryset(self, request, queryset, view):
         queryset = queryset.extra(facets=facets)
 
         return queryset
+
+
+class HighlightFilterBackend(BaseFilterBackend):
+    """
+    A django-rest-framework filter backend that adds search term highlighting.
+
+    If the query string `highlight=0` is passed, we bypass highlighting.
+
+    """
+    def filter_queryset(self, request, queryset, view):
+        highlight = request.GET.get('highlight', True)
+        if highlight != '0':
```

Comment:
I dislike GET parameters that assume that 0 is false-y, let's make that `?highlight=false`